test/studies/shootout/meteor/kbrady/meteor-parallel.chpl

```diff
@@ -532,25 +537,22 @@ module meteor {
 
 
   proc solve() {
-    forall piece in 0..9:uint(8) {
+    forall piece in piecesDom do
       solve_helper(piece);
-    }
   }
 
-  proc solve_linear(in depth: int, in cell: int, in board: uint,
-      in avail: uint(16), solNums: [] uint(8), solMasks: [] uint) {
-    var pieceNoMask: uint(16);
-    var maxRots: int;
-    var pieceMask: uint;
+  proc solve_linear(in depth, in cell, in board,
+      in avail, solNums, solMasks) {
+    var pieceNoMask, maxRots, pieceMask: int;
 
     if solutionCount.read() >= maxSolutions then
       return;
 
     while (board & (1 << cell)) do
       cell += 1;
 
-    for piece in 0..9:uint(8) {
-      pieceNoMask = 1:uint(16) << piece;
+    for piece in piecesDom {
+      pieceNoMask = 1 << piece;
       if !((avail & pieceNoMask):bool) {
```

Comment:
It seemed to work just fine.  However, I'm realizing that our performance testing doesn't fully diff the output for this test.  We get the same number of solutions, which means we're probably fine, but that doesn't mean the last solution found is correct.  I will verify that change works through the shootout script before committing it (and will definitely verify the rest of it works before submitting to the site).